packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -314,14 +320,27 @@ const AccountTransferForm = observer(
             };
 
             remaining_transfers = getRemainingTransfers();
-
-            const hint =
-                remaining_transfers && Number(remaining_transfers) === 1
-                    ? localize('You have {{number}} transfer remaining for today.', { number: remaining_transfers })
-                    : localize('You have {{number}} transfers remaining for today.', { number: remaining_transfers });
-            setTransferToHint(hint);
+            let hint_text;
+            // flag 'open_position_status' does not exist in mt5_login_list, @deriv/api-types yet
+            if (mt5_login_list.find(account => account?.login === selected_to.value)?.open_position_status) {
+                hint_text = <Localize i18n_default_text='You can no longer open new positions with this account.' />;
+            } else {
+                hint_text =
+                    remaining_transfers && Number(remaining_transfers) === 1 ? (
+                        <Localize
```

Comment:
Can u handle pluralization of text `transfer` using a variable `remaining_transfers` rather than having 2 Localize?